### ISSUE TITLE
multiple cards toggling not working

### ISSUE THREAD

**Issue by necmes** [PiotrMachowski/Home-Assistant-Lovelace-Local-Conditional-card#5]
Hello,

First of all thanks for this awesome card!

I create this issue because i'am unable to toggle 2 entities at once.

my card config:
```
type: vertical-stack
cards:
  - type: 'custom:local-conditional-card'
    id: player1
    default: show
    card:
      entities:
        - media_player.office
      title: Office
      type: entities
  - type: 'custom:local-conditional-card'
    id: player2
    default: hide
    card:
      entities:
        - media_player.kitchen
      title: Kitchen
      type: entities
  - title: Click test
    type: entities
    entities:
      - action_name: Toggle
        name: toggle 1 and 2
        service: local_conditional_card.toggle
        service_data:
          ids:
            - player1
            - player2
        type: call-service
```

player Kitchen is toggling but not player Office

if i'll swap player1 and 2 like this:
```
    entities:
      - action_name: Toggle
        name: toggle 1 and 2
        service: local_conditional_card.toggle
        service_data:
          ids:
            - player2
            - player1
        type: call-service
```

Same behaviour only the second entity is toggling.

if i swap entities on card list:
```
cards:
  - type: 'custom:local-conditional-card'
    id: player1
    default: show
    card:
      entities:
        - media_player.kitchen
      title: Office
      type: entities
  - type: 'custom:local-conditional-card'
    id: player2
    default: hide
    card:
      entities:
        - media_player.office
      title: Kitchen
      type: entities
```
The player Office is now toggling but not the player Kitchen

It's like the Toggling function only use the ID of the last defined entity on cards list.

**Comment by andiukas** [comment]
the solution is?

**Comment by TEF2one** [comment]
Sadly seem over a year and still having the same issue...

**Comment by PiotrMachowski** [comment]
@TEF2one you are right, after 2 years of not being updated this card definitely deserves some love. `v2.0.0` coming really soon!

**Comment by PiotrMachowski** [comment]
Fixed in v2.0.0
